docs/source/format/FlightSql/CommandStatementQuery.mmd

```diff
@@ -0,0 +1,28 @@
+%% Licensed to the Apache Software Foundation (ASF) under one
+%% or more contributor license agreements.  See the NOTICE file
+%% distributed with this work for additional information
+%% regarding copyright ownership.  The ASF licenses this file
+%% to you under the Apache License, Version 2.0 (the
+%% "License"); you may not use this file except in compliance
+%% with the License.  You may obtain a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+
+sequenceDiagram
+autonumber
+
+participant Client
+participant Server
+Client->>Server: GetFlightInfo(CommandStatementQuery)
+Server->>Client: FlightInfo{…Ticket…}
+loop for each Ticket in FlightInfo
```

Comment:
Is this the case? I though all returned tickets were equivalent and only one of them needed to be fetched, am I misremembering?